quorum/templates/08-epirus-chain-explorer.yaml

```diff
@@ -0,0 +1,61 @@
+{{ $scope := . }}
+{{- if eq $scope.Values.epirus.enabled true }}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: epirus-chain-explorer-deployment
+spec:
+  strategy:
+    type: Recreate
+  replicas: 1
+  selector:
+    matchLabels:
+      name: epirus-chain-explorer-deployment
+  template:
+    metadata:
+      name:  epirus-chain-explorer-deployment
+      labels:
+        app: epirus
+        tier: backend
+        name: epirus-chain-explorer-deployment
+    spec:
+      securityContext:
+      containers:
+      - name: epirus-api
+        image: web3labs/epirus-free-api:latest
+        ports:
+          - containerPort: 8090
+        env:
+          - name: NODE_ENDPOINT
+            value: http://{{$scope.Values.epirus.node}}.quorum-network.svc.cluster.local:8545
```

Comment:
This includes the namespace in a hard-coded way, I think if we assume that Epirus is running in the same namespace we don't need the `.<namespace>.svc.cluster.local` suffix

Reply:
removed the suffix, works well